### PR TITLE
feat(audit_logs): add hanlders

### PR DESCRIPTION
**Pull request by qtotuan**

### CODE REVIEW

File: src/tillhub-js.ts [hunk 1]
```diff
@@ -435,6 +435,24 @@ export class TillhubClient extends events.EventEmitter {
     return new v0.AuditActions({ user: this.auth.user, base: this.options.base }, this.http)
   }
 
+  /**
+   * Create an authenticated Staff instance
```
Comment (erezsob) [inline src/tillhub-js.ts]: Change comment to be audit logs related

File: src/v0/auditLogs.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      try {
+        if (q && q.uri) {
+          uri = q.uri
+        } else {
+          uri = `${this.options.base}${this.endpoint}/${this.options.user}/logs`
+        }
+
+        const queryString = qs.stringify(q)
+        if (queryString) {
+          uri = `${uri}?${queryString}`
```
Comment (erezsob) [inline src/v0/auditLogs.ts]: As we talked F2F, please use the `UriHelper` class to construct the uri.

File: src/tillhub-js.ts (hunk 1 above)
Comment (qtotuan) [reply src/tillhub-js.ts]: Changed

File: src/v0/auditLogs.ts (hunk 2 above)
Comment (qtotuan) [reply src/v0/auditLogs.ts]: I refactored it

**Pull request merged**